src/components/uielements/input/Input.tsx

```diff
@@ -2,22 +2,28 @@ import React from 'react'
 
 import { InputProps } from 'antd/lib/input'
 
-import { InputWrapper } from './Input.style'
-import { Color, InputType, Size } from './types'
+import { InputWrapper, InputPasswordWrapper } from './Input.style'
```

Comment:
Why do we need this `Input` component? It just wrapped `InputPasswordWrapper` / `InputWrapper` and both could be imported into any other component it if needed.

Reply:
I guess that's true.. The Input component already existed in the project so I just extended on it.

Reply:
Resolved with b6bbb8d135cffd61aab07a9560c66ee04c629085